Zend/tests/bug76337.phpt

```diff
@@ -1,9 +1,7 @@
 --TEST--
 Bug 76337: segmentation fault when an extension use zend_register_class_alias() and opcache enabled
 --SKIPIF--
-<?php
-if (!extension_loaded('Zend OPcache')) die('skip Zend OPcache extension not loaded');
-if (!class_exists('_ZendTestClass')) die('skip _ZendTestClass class does not exist');
+<?php if (!extension_loaded('zend-test')) die('skip zend-test extension not loaded');
```

Comment:
I believe the SKIPIF still should have the opcache check here, perhaps we should move it into the right place (ext/opcache/tests)

Reply:
Move to `ext/opcache/tests` sounds great, but, adding the Zend OPCache extension check sounds overwork, as it by itself, won't make the test run :(

Reply:
Sure, but it does show us the condition for where the bug occured, I agree its not a common build but I don't mind such exceptions personally. Either way is fine with me tbh